scripts/updateAutocompleteDocs.js

```diff
@@ -11,14 +11,18 @@ const SRC_FILENAME = "global.html"
 const OUT_DIR = "src/editors/ace/docTooltip"
 
 const TARGETS = [
-  "source_1",
-  "source_1_wasm",
-  "source_2",
-  "source_3",
-  "source_3_concurrent",
-  "source_3_non-det",
-  "source_4",
-  "External libraries",
+  'source_1',
+  'source_1_wasm',
+  'source_1_typed',
+  'source_2',
+  'source_2_typed',
+  'source_3',
+  'source_3_concurrent',
+  'source_3_non-det',
+  'source_3_typed',
+  'source_4',
+  'source_4_typed',
+  'External libraries'
```

Comment:
Seems like your editor is changing double quotes to single quotes automatically.... Hmm, I wonder why the CI allows this. Could you just change the necessary lines?